main/src/cgeo/geocaching/maps/LivemapStrategy.java

```diff
@@ -9,7 +9,6 @@
  * Defines the strategy for the Live Map
  */
 public enum LivemapStrategy {
-    FASTEST(1, EnumSet.of(Flag.LOAD_TILES), R.string.map_strategy_fastest),
```

Comment:
Yes, I'll add a comment here. First I thought to keep the commented out code there, but decided against it, because it is bad practice. But a comment that there was another option which is not supported anymore would be good.